distribution/src/main/packaging/scripts/postrm

```diff
@@ -76,6 +77,12 @@ if [ "$REMOVE_DIRS" = "true" ]; then
         echo " OK"
     fi
 
+    if [ -d "$MODULES_DIR" ]; then
```

Comment:
I feel like we should be able to mark the directory in the same way that we mark the `lib` and rpm/deb should handle that for us?